src/components/notebook-utils.js

```diff
@@ -100,32 +102,34 @@ export class NotebookCreator extends Component {
           () => h(Modal, {
             onDismiss: () => this.setState({ modalOpen: false }),
             title: 'Create New Notebook',
-            okButton: buttonPrimary({
-              disabled: errors,
-              onClick: () => {
-                this.setState({ modalOpen: false, creating: true })
-                Buckets.notebook(namespace, bucketName, notebookName).create(notebookKernel.data).then(
-                  () => {
-                    this.setState({ creating: false })
-                    reloadList()
-                  },
-                  error => {
-                    this.setState({ creating: false })
-                    reportError('Error creating notebook', error)
-                  }
-                )
-              }
-            }, 'Create Notebook')
+            okButton: h(TooltipTrigger, { content: Utils.summarizeErrors(errors) }, [
+              buttonPrimary({
+                disabled: errors,
+                onClick: () => {
+                  this.setState({ modalOpen: false, creating: true })
+                  Buckets.notebook(namespace, bucketName, notebookName).create(notebookKernel.data).then(
+                    () => {
+                      this.setState({ creating: false })
+                      reloadList()
+                    },
+                    error => {
+                      this.setState({ creating: false })
+                      reportError('Error creating notebook', error)
+                    }
+                  )
+                }
+              }, 'Create Notebook')
+            ])
           }, [
-            div({ style: Style.elements.sectionHeader }, 'Name'),
+            div({ style: Style.elements.sectionHeader }, 'Name *'),
```

Comment:
I may be misremembering, but I believe we decided to mark only optional inputs app-wide?

Reply:
There are definitely recent designs showing asterisks for required. Let's follow up with design.